proposals/thread-spawn/Overview.md

```diff
@@ -1,11 +1,13 @@
 # [DRAFT] Thread Spawn Proposal
 
 This page describes a proposal to allow WebAssembly modules to spawn threads from within the
-WebAssembly language. This is done with three additions to the specification:
+WebAssembly language. This is done with several additions to the specification (each addition is
+annotated with its criticality to the proposal; next-stage discussions could alter this list):
 
-- a new `thread.spawn` instruction
-- new `shared` attributes on WebAssembly tables, functions, and globals
-- a new `thread.hw_concurrency` instruction
+- __required__: new `shared` attributes on WebAssembly tables, functions, and globals
+- __required in some form__: a new `thread.spawn` instruction (other mechanisms are possible; more
```

Comment:
would mention here - some kind of facilitation of thread-local storage - we're pitching `thread.id`, the design of which is heavily linked to the design of `thread.spawn`

Reply:
Yeah, good idea. I added a line for that in my latest commit. I also started thinking through whether we need to highlight the "end a thread" idea here, e.g., a handle for terminating a thread from outside and/or a new instruction like `thread.exit` to force an early exit. But we can add that later, I guess.